scripts/mwsitemapgen.sh

```diff
@@ -33,6 +33,10 @@ while true; do
       --identifier=mediawiki \
       >> "$logfileNow" 2>&1
 
+    # sending the sitemap to google
+    echo "sending to Google -> https://www.google.com/ping?sitemap=$WG_SITE_SERVER/$WG_SCRIPT_PATH/sitemap/sitemap-index-mediawiki.xml"
```

Comment:
I think we don't need `/` before $WG_SCRIPT_PATH because it already starts with `/`. (slash is doubled now)
Also $WG_SITE_SERVER can be a protocol-relative url like `//site.name`